src/pages/components/anchor.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Anchor
+description: >
+  The Anchor component is a link used to navigate from one location to another.
+package: '@zendeskgarden/react-buttons'
+propSheets:
+  - buttons/src/elements/Anchor.tsx
+---
+
+import AnchorDefault from '../../examples/anchor/AnchorDefault.tsx';
+import AnchorDefaultCode from '!!raw-loader!../../examples/anchor/AnchorDefault.tsx';
+import AnchorExternal from '../../examples/anchor/AnchorExternal.tsx';
+import AnchorExternalCode from '!!raw-loader!../../examples/anchor/AnchorExternal.tsx';
+import AnchorDanger from '../../examples/anchor/AnchorDanger.tsx';
+import AnchorDangerCode from '!!raw-loader!../../examples/anchor/AnchorDanger.tsx';
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To navigate from page to page</li>
+      <li>To display links inline with text</li>
+      <li>To navigate within a page</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        To prompt an action or submit data, use a <a href="/components/button">Button</a>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## Examples
+
+### Default
+
+An Anchor is used as a link to signal navigation.
+
+<CodeExample code={AnchorDefaultCode}>
+  <AnchorDefault />
+</CodeExample>
+
+### External Anchor
+
+Shows an icon at the end of the link and opens it in a new window or tab.
```

Comment:
The `isExternal` prop doesn't actually open the link in a new window or tab. That's dependent on the `target` attribute. This wording may need to be tweaked.

Reply:
Good callout. Is there a reason why we don't make that functionality part of the prop?

Reply:
> Is there a reason why we don't make that functionality part of the prop?

Good question 🤔. I think it's because `target` is a native `<a>` attribute. So no need for a prop, one can simply pass the `target` attribute directly. However, showing an icon is not native behavior, so we use a prop to customize and allow for rendering an icon.

Reply:
It's difficult to infer that a specific `target` value is always "external" as well. Depending on where the link is shown (like in an iframe) it's not something that we can reliably enforce.

Reply:
It is so great we're uncovering these things. These are improvements to the code and communication that I think will really help adoption.